src/main/java/com/solvd/elasticsearchmicroservice/kafka/KfConsumerImpl.java

```diff
@@ -1,38 +1,36 @@
 package com.solvd.elasticsearchmicroservice.kafka;
 
-import com.solvd.elasticsearchmicroservice.domain.Note;
 import com.solvd.elasticsearchmicroservice.kafka.event.NoteEvent;
-import com.solvd.elasticsearchmicroservice.service.NoteService;
+import com.solvd.elasticsearchmicroservice.kafka.handler.DeleteHandler;
+import com.solvd.elasticsearchmicroservice.kafka.handler.Handler;
+import com.solvd.elasticsearchmicroservice.kafka.handler.PostHandler;
+import com.solvd.elasticsearchmicroservice.kafka.handler.PutHandler;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.stereotype.Service;
 
+import java.util.HashMap;
+import java.util.Map;
+
 @Slf4j
 @Service
 @RequiredArgsConstructor
 public class KfConsumerImpl implements KfConsumer{
 
-    private final NoteService noteService;
+    private final PostHandler postHandler;
+    private final PutHandler putHandler;
+    private final DeleteHandler deleteHandler;
+
+    private final Map<String, Handler> processors = new HashMap<>();
```

Comment:
@vpetrovaa Unbreakable dependency here

It will be very hard (almost impossible) to test this part of `KfConsumer`